### DISCUSSION TITLE
Requesting Rony Shapiro's Password Safe but without admin privileges

### DISCUSSION THREAD

**Discussion by southisup** [microsoft/winget-pkgs#158051] [Q&A] [answered]
Hi
I used the Update Request template to request Rony Shapiro's Password Safe be changed: https://github.com/microsoft/winget-pkgs/issues/157627
But instead of the desired local user privileges extractable EXE option, it's resulted in an MSI, which require admin privileges, and I am still unable to replace* my old version with a current one.

Do I need to make the request using a different template, or is what I'm requesting not possible when there's an MSI available?

`ExifTool_is1` is an example of something I am able to update myself without admin.

*I understand that that will in fact not actually be uninstalled, and it will be on me to remember to use the separate, current version, not the superseded one.

**Comment by SpecterShell** [comment, marked as answer]
The EXE installer couldn't be added because it has a language selector that would pop up even in silent mode and thus block silent installation. It should be related to <https://github.com/pwsafe/pwsafe/blob/b04dfa3b75a31ad110ff3da550a0680a574b6b7f/install/windows/pwsafe.nsi#L982>, which could be bypassed by using `IfSilent`. You can report this issue to the author.

(You can also ask the author to switch to the UTF-8 version of NSIS if possible. The installer text garbled on my device).

**Reply by southisup** [reply]
Thanks! Enhancement requested:
https://sourceforge.net/p/passwordsafe/feature-requests/924/
